applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -111,7 +187,7 @@ static int lc3_cap_reconfig_cb(struct bt_audio_stream *stream, struct bt_audio_c
 	LOG_INF("ASE Codec Reconfig: stream %p cap %p", (void *)stream, (void *)cap);
 
 	/* We only support one QoS at the moment, reject changes */
-	return -ENOEXEC;
+	return 0;
```

Comment:
If we change the return value in here, probably means we can remove above comment.
May I know how to test the "reconfig"?

Reply:
Yes, will remove. It happens if you have a stream and reset gateway